### ISSUE TITLE
Tauri v2 && Astro 4?

### ISSUE THREAD

**Issue by somethingxdev** [miguelgargallo/tauri-astro-app#3]

**Comment by miguelgargallo** [comment]
I don't understand your question, can you rephrase? Thanks

**Comment by somethingxdev** [comment]
> I don't understand your question, can you rephrase? Thanks

I wanted to clarify if you will update the repository based on Astro v4 and the new Tauri v2?

**Comment by miguelgargallo** [comment]
> anted to clarify if you will update the repository based on Astro v4 and the ne

Yeah!, it's on the weekend roadmap!

**Comment by miguelgargallo** [comment]
But any PR @somethingxdev will be accepted to!!
